### DISCUSSION TITLE
Welcome to Gutenberg Discussions!

### DISCUSSION THREAD

**Discussion by gziolo** [WordPress/gutenberg#28718] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fabiankaegy** [comment]
Hey @gziolo, I've hoped for sooo long that this repo gets the Discussions tab because I just think it makes it easier to distinguish between issues and discussions. So thanks for adding it :) 

I'm Fabian 👋 I currently work @10up as a Frontend Engineer and therefore use Gutenberg on a daily basis to build editorial experiences for clients. Both through my role there but also in my free time I try to give back as much as I can with testing or working on documentation fixes, sometimes triaging tickets and sometimes code contributions.

**Comment by eirichmond** [comment]
I'm Elliott, and I've been in the WordPress game for as long as WordPress has been around. Yep, nearly 2—decades!
I've been building themes and plugins, helping clients with custom WordPress solutions, and diving deep into open source for what feels like forever. I also run meetups in Cheltenham, UK, bringing local devs and users together.
More recently, I've been getting into the education side of things, sharing what I've learned over the years on my YT channel, with the help of the community I hope to increase and improve on the quality of my content.
WordPress has been my jam for so long, and I'm itching to get more involved in the community here on GitHub. 
Cheers,
Elliott
